kpops/components/streams_bootstrap/producer/producer_app.py

```diff
@@ -26,6 +26,7 @@ class ProducerApp(KafkaApp):
         default="producer", exclude=True
     )
     app: ProducerValues
+    from_: None = None
```

Comment:
Shouldn't that be of type `FromSection`?

Reply:
we want to disallow setting a `FromSection` for producers, because it doesn't support it